pom.xml

```diff
@@ -240,20 +260,42 @@
         <defaultGoal>clean install</defaultGoal>
         <finalName>sdx-gateway</finalName>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M3</version>
```

Comment:
Likewise, can we remove the milestone version and use a released jar

Reply:
Like the surefire plugin above, v3.x of enforcer plugin has only been on milestone releases since 2017:
http://archive.apache.org/dist/maven/surefire/

But it builds fine here with 1.4.1, and this code isn't actually using enforcer to `requireJavaVersion`: it's using it to `requireMavenVersion`, to get maven to the level needed for the `maven-compiler-plugin` update needed for J11. But I've just changed "3.3.9" to "3.3," to allow a less restrictive maven 3.3 or higher.